Jenkinsfile

```diff
@@ -84,3 +83,30 @@ demos['databound'] = {
 }
 
 parallel(demos)
+
+node('docker') {
+    def image
+    def imageName = 'jenkins/jenkinsfile-runner-experimental'
+    def imageTag
+
+    stage('Checkout') {
+        timestamps {
+            deleteDir()
+            checkout scm
+
+            sh 'git rev-parse HEAD > GIT_COMMIT'
```

Comment:
If you store the result from "checkout scm", the git commit is something you can ask to that (see https://jenkins.io/doc/pipeline/steps/workflow-scm-step).

Reply:
thanks!